packages/storybook/cxl-ui/cxl-vaadin-accordion/cxl-playbook-accordion.story.js

```diff
@@ -6,15 +6,24 @@ export const CXLPlaybookAccordion = () => html`
   <cxl-playbook-accordion id="playbook-3895" class="plural" opened="0">
     ${playbookStepData.map(
       (el) => html`
-        <vaadin-accordion-panel class="playbook-step" theme="cxl-playbook-accordion reverse">
+        <vaadin-accordion-panel
+          class="playbook-step"
+          theme="cxl-playbook-accordion reverse"
+          data-step-id="${el.id}"
+        >
           <header class="entry-header" slot="summary">
             <h3 class="entry-title no-anchor" itemprop="headline">
               <vaadin-checkbox value="${el.id}"></vaadin-checkbox>
               ${el.id}. ${unsafeHTML(el.title.rendered)}
             </h3>
           </header>
-
           <div class="entry-summary" itemprop="description">${unsafeHTML(el.content.rendered)}</div>
+          <div class="entry-footer">
+            <vaadin-button class="feedback-button">
```

Comment:
Can also drop invented class here. We'll figure out separation when something else comes to live inside `.entry-footer`, until then it's simple enough to select with just `.entry-footer > vaadin-button`.

Reply:
done